Specs/DateTime/Japanese/DatePeriodParser.json

```diff
@@ -5373,41 +5212,15 @@
       }
     ]
   },
-  {
-    "Input": "２００８年　十二月",
```

Comment:
Does this case exist anywhere else?

Reply:
added back (space was removed because normally it is not used between those characters, but it is not wrong to have it)